src/Grpc/JsonTranscoding/perf/Microsoft.AspNetCore.Grpc.Microbenchmarks/Microsoft.AspNetCore.Grpc.Microbenchmarks.csproj

```diff
@@ -6,6 +6,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TieredCompilation>false</TieredCompilation>
     <DefineConstants>$(DefineConstants);IS_BENCHMARKS</DefineConstants>
+    <SkipMicrobenchmarksValidation>true</SkipMicrobenchmarksValidation>
```

Comment:
What is this change for?

Reply:
The microbenchmark fails to find the just built SharedFx - this happens every time we update the branding/TFM in certain microbenchmarks, we're able to re-enable it once we get an SDK with the new TFM

Reply:
Ok. Maybe add a comment?

Reply:
https://github.com/dotnet/aspnetcore/issues/57461